chef/cookbooks/keystone/attributes/default.rb

```diff
@@ -40,6 +40,7 @@
 default[:keystone][:api][:admin_host] = "0.0.0.0"
 default[:keystone][:api][:api_host] = "0.0.0.0"
 default[:keystone][:api][:version] = "v2.0"
+default[:keystone][:api][:version_multidomain] = "v3"
```

Comment:
I don't think this is the right approach for introducing multidomain support. First the V3 API can be used of much more things that just multi domain (e.g. groups, finer grained admin policies, etc).

 And then, when enabling the use of multiple domains all services need to be switch to use the v3 API by default (nova, neutron etc...) in that case it would make sense to switch ```[:keystone][:api][:version]``` to ```v2.0``` I think. There is not much use left for the v2.0 API when using a multi domain setup or am I missing something?